.github/workflows/tests.yml

```diff
@@ -90,6 +90,13 @@ jobs:
           echo "DASK_DATAFRAME__QUERY_PLANNING: $DASK_DATAFRAME__QUERY_PLANNING"
           echo "DASK_DATAFRAME__QUERY_PLANNING=$DASK_DATAFRAME__QUERY_PLANNING" >> $GITHUB_ENV
 
+      - name: Disable auto pyarrow strings for 3.10 builds
+        if: ${{ matrix.environment == '3.10' }}
+        run: |
+          export DASK_DATAFRAME__CONVERT_STRING="False"
+          echo "DASK_DATAFRAME__CONVERT_STRING: $DASK_DATAFRAME__CONVERT_STRING"
+          echo "DASK_DATAFRAME__CONVERT_STRING=$DASK_DATAFRAME__CONVERT_STRING" >> $GITHUB_ENV
+
```

Comment:
Previously our mindeps build had an old enough `pyarrow` that auto pyarrow strings wasn't supported. This is no longer the case with the new `pyarrow=14.0.1` minimum version, so I'm explicitly disabling `convert_string` in our Python 3.10 builds so we keep that test coverage (similar to what we're also doing with query planning).